BuildMasterAutomation/Functions/Invoke-BMRestMethod.ps1

```diff
@@ -159,9 +159,11 @@ function Invoke-BMRestMethod
     {
         if( $Method -eq [Microsoft.PowerShell.Commands.WebRequestMethod]::Get -or $PSCmdlet.ShouldProcess($Uri,$Method) )
         {
+            Write-Debug "$($webRequestParam.ToString())"

```

Comment:
Left-over debugging statement. Remove or improve it so that it includes all the information about the request: Url, Method, Headers, Body.